### DISCUSSION TITLE
What causes a workflow manual dispatch to hang endlessly when bringing up the branch/input modal?

### DISCUSSION THREAD

**Discussion by apearce-nov** [community/community#42167] [Actions] [answered]
### Select Topic Area

Question

### Body

I have multiple workflows that have manual dispatch events setup that have been working fine. Out of nowhere, they all hang endlessly when trying to bring up the input modal. Is this a github issue or a problem with my codebase?

https://user-images.githubusercontent.com/74783074/208553978-1d21039b-e338-4e49-8fc8-90ac3ab8f414.mov

**Comment by jamesloosli** [comment]
I'm seeing this as well.

I'm going to guess there's some kind of outage?

**Reply by apearce-nov** [reply]
Thanks for the quick reply. Wanted to make sure I wasn't going crazy because I've been making a lot of changes to the manual workflows

**Reply by jamesloosli** [reply]
Yours was the first discussion when I landed on the page. Can't be the only ones.

**Comment by darkcl** [comment]
same for my repository, i see 404 when i trigger the modal :thinking: 

![Screenshot_2022-12-20_08-46-18](https://user-images.githubusercontent.com/446404/208555742-f6e904fe-7591-46a2-88ff-3e9d8e6c1af3.png)

**Comment by jamesloosli** [comment, marked as answer]
https://github.com/orgs/community/discussions/42166
https://github.com/community/community/discussions/42164

Not just us, it seems.

https://www.githubstatus.com/incidents/3y3y8nnbyt2d
